src/js/components/sharedComponents/CanopyDensityPicker.tsx

```diff
@@ -70,34 +71,36 @@ const CanopyDensityPicker = (props: CanopyDensityProps): JSX.Element => {
   }
 
   const handleClick = async (type: 'meter' | 'hectare') => {
-    const layerConfig = allAvailableLayers.find((layer: any) => layer.id === 'TREES_MOSAIC_LANDSCAPES') as any;
```

Comment:
@anthonyamaro15 Looks like you may overwrote some of the TREES_MOSAIC_LANDSCAPES code we may need to revert due to use keeping TREES_MOSAIC_LANDSCAPES until we deprecate.

Reply:
@anthonyamaro15 Can you look throughout the code and ensure we are not messing with TREES_MOSAIC_LANDSCAPES